plugin/adsk/scripts/mayaUsdTranslatorExport.mel

```diff
@@ -749,8 +806,11 @@ global proc int mayaUsdTranslatorExport (string $parent,
         string $expandedSections[];
         string $collapsedSections[];
         string $sectionNames[] = parseActionSectionNames($sections, $expandedSections, $collapsedSections);
+        // collect all exclude types before append to option list
+        string $excludeExportTypes = "";
             
         if (stringArrayContains("geometry", $sectionNames)) {
+            $excludeExportTypes = mayaUsdTranslatorExport_SetSuboptionsFromCheckBox($excludeExportTypes, "", "Meshes", "exportMeshesCheckBox");

```

Comment:
The flag is passed explicitly here, so changing from "Meshes" to "mesh" seem feasible without modifying UI shown to teh user.

Reply:
I don't know if that will confuse the user.. Like they see "Meshes" in the UI but when use it in command it changes to "mesh". They won't know it unless they read extra documents to get that information...

Reply:
But yea its definitely feasible